torchci/test/acceptBot.test.ts

```diff
@@ -76,4 +76,40 @@ describe("accept bot", () => {
 
     handleScope(scope);
   });
+
+  test("Run when PR is accepted already and then labeled", async () => {
+    const payload = requireDeepCopy("./fixtures/pull_request.labeled.json");
+    payload.label.name = ACCEPT_2_RUN;
+
+    const owner = payload.repository.owner.login;
+    const repo = payload.repository.name;
+    const pr_number = payload.pull_request.number;
+    const scope = nock("https://api.github.com")
+      .get(`/repos/${owner}/${repo}/pulls/20/reviews`)
+      .reply(200, [{ state: "approved" }])
+      .post(`/repos/${owner}/${repo}/issues/20/labels`)
+      .reply(200, {})
+      .delete(`/repos/${owner}/${repo}/issues/20/labels/${ACCEPT_2_RUN}`)
+      .reply(200, {});
+
+    await probot.receive({ name: "pull_request", id: "123", payload });
+
+    handleScope(scope);
```

Comment:
This comment is more about my own naivete

Where does the actual testing happen in here?  Is it the `probot.receive` command where the "webook" is actually received and acted upon? 

If so, what does `handleScope` do?

Reply:
handlescope just asserts that all of the mocks to the scope on line 87 are satisfied and that no extra calls are made. It essentially just checks to see once we receive this webhook event, do we make the corresponding API calls to act up on the event.